src/lib/api.js

```diff
@@ -7,10 +7,9 @@
  * External dependencies
  */
 import fetch from 'node-fetch';
-import { ApolloClient } from 'apollo-client';
-import { HttpLink } from 'apollo-link-http';
-import { InMemoryCache } from 'apollo-cache-inmemory';
-import { onError } from 'apollo-link-error';
+import { ApolloClient, ApolloLink, HttpLink } from '@apollo/client/core';
+import { InMemoryCache } from '@apollo/client/cache';
+import { onError } from '@apollo/client/link/error';
```

Comment:
It seems like with every version, they flip back and forth between separate and a mono package 😆

Reply:
This is the way.